src/js/store/tasks/selectors.ts

```diff
@@ -35,10 +42,10 @@ export const selectTasksByEpic = createSelector(
     /* istanbul ignore else */
     if (epic && tasks[epic.project]?.fetched) {
       const fetched = tasks[epic.project].fetched;
-      if (fetched === true || fetched.includes(epic.id)) {
+      if (fetched.includes(epic.id)) {
         return {
           tasks: filter(tasks[epic.project].tasks, ['epic.id', epic.id]),
-          [epic.id]: tasks.next,
+          next: tasks[epic.project].next[epic.id],
```

Comment:
These were the other notable changes -- now it's returning a `next` string that indicates whether there is a next page for the given epic.